docs/client/apidocs.rst

```diff
@@ -0,0 +1,110 @@
+API Reference
+=============
+
+Module contents
+---------------
+
+.. automodule:: scrapinghub.client
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+Submodules
+----------
+
+activity
+--------
+
+.. automodule:: scrapinghub.client.activity
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
also ideally `list()` on the above screenshot should reference proper list method in the Activity class